common/djangoapps/third_party_auth/saml.py

```diff
@@ -211,6 +211,17 @@ def get_user_details(self, attributes):
         })
         return details
 
+    def get_attr(self, attributes, conf_key, default_attribute):
```

Comment:
i'm not seeing where this function gets called...
also, is it possible to rename it to something more specific? it could get confused very easily with getattr

Reply:
This method is defined in the superclass [SAMLIdentityProvider](https://github.com/python-social-auth/social-core/blob/1.4.0/social_core/backends/saml.py#L68), I am overriding it here so, it can't be renamed.